aten/src/ATen/native/cuda/group_norm_kernel.cu

```diff
@@ -216,9 +216,9 @@ __global__ void GroupNormBackwardSimpleCUDAKernel(
 
 template <typename T>
 __global__ void GroupNormBackwardCUDAKernel(
-    int64_t C,
-    int64_t HxW,
-    int64_t group,
+    const int64_t C,
+    const int64_t HxW,
+    const int64_t group,
```

Comment:
Is this const required?

Reply:
They are not strictly required. I assume setting them to `const` may enable better compiler optimizations.

Reply:
I think this may not generate relative large differences. From the code style point of view, I'd suggest to make them same as others. But it is fine to keep them.

Reply:
I agree. Thanks!

Reply:
Const in the argument list doesn't matter for optimizations. If you don't store into the variable the compiler should be easily able to view it. It is mostly a style thing (to prevent accidental reassignment).